### ISSUE TITLE
Number in BREAD does not accept decimals

### ISSUE THREAD

**Issue by hotsaucejake** [thedevdojo/voyager#1181]
- Laravel Version: 5.4.19
- Voyager Version: 0.11.9
- PHP Version: 7.1
- Database Driver & Version:

### Description:
I have a column in a table that is a decimal (with a specific length).  Creating my BREAD I select "Number" as the type.  Once the bread is created there is a validator on the frontend (before submission) that does accept decimal values.  It only allows me to input integers.  Once I submit an integer, a decimal is represented in the DB.  The DB is performing correctly, it's the validator on the frontend of the BREAD that doesn't allow for decimals.

![voyager_decimal](https://cloud.githubusercontent.com/assets/6332533/25441558/910a51c6-2a70-11e7-9535-98073896c846.png)

### Steps To Reproduce:

* make column in table a decimal
* create BREAD, select "Number" as format
* try to enter a decimal number on your newly created BREAD

**Comment by hotsaucejake** [comment]
Referencing this article: https://www.isotoma.com/blog/2012/03/02/html5-input-typenumber-and-decimalsfloats-in-chrome/

I found a solution.  Where the HTML is rendered to this:
````html
<input type="number" class="form-control" name="amount" placeholder="Amount" value="999.99">
````

Simply add another attribute to the input ````step="any"```` like so:

````html
<input type="number" step="any" class="form-control" name="amount" placeholder="Amount" value="999.99">
````

**Comment by hotsaucejake** [comment]
Finally, change to be made here: https://github.com/the-control-group/voyager/blob/master/resources/views/formfields/number.blade.php

**Comment by endrureza** [comment]
@thedevdojo perhaps the artist could notice about this

**Comment by hotsaucejake** [comment]
@marktopper and @davodavodavo3 committed to that blade specifically.  

I went ahead and added the change to the blade on my end instead of waiting on an update and it works fine now!  If anybody else if having trouble, just find the file above and insert ````step="any"```` and all will be good with the world.

**Comment by fletch3555** [comment]
@hotsaucejake, I'm okay with that change.  If you'd like to make the change and open a Pull Request for it, I'd be more than happy to let you have credit for it! Good find!

This is definitely the correct fix.  For reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number.  Perhaps we could make the "step" value configurable in the BREAD configuration for that field, and it could just default to "any".

**Comment by hotsaucejake** [comment]
Submitted the pull request.  A configurable step would be ideal in the future.

#1190

**Comment by fletch3555** [comment]
Closed by #1190
